pb-crm-ui/src/app/contact-book/components/contact-book-home/contact-book-home.component.ts

```diff
@@ -38,10 +39,33 @@ export class ContactBookHomeComponent implements OnInit, AfterViewInit {
     private dialog: MatDialog,
     private contactCache: ContactCacheService,
     private snackbar: MatSnackBar,
-    private viewContainerRef: ViewContainerRef
+    private viewContainerRef: ViewContainerRef,
+    private dialogService: DialogService,
   ) {
   }
 
+  openImportDialog(): void {
+    const importDialogData: DialogInterface = {
+      title: 'Import Contact File',
+      showSubmitBtn: false,
+      showOkBtn: true,
+      showCancelBtn: true,
+      component: ImportFileComponent
+    };
+
+    const dialogRef = this.dialogService.openDialog(
+      importDialogData, {  });
+
+    dialogRef.afterClosed().subscribe(result => {
+      if (result) {
+        // do something with ok/submit
+      } else {
+        // cancel / close dialog
+        console.log('User clicked cancel');
+      }
+    });
```

Comment:
This seems like its not yet completed. Is this what you want here?

Reply:
definitely not completed, still needs to pass a file to the back.